docs/getting-started-guides/kubeadm.md

```diff
@@ -473,8 +473,10 @@ proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/mu
 deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out
 if there's interest).
 
-Currently, only the pod network flannel is working on multiple architectures.
-You can install it this way:
+Currently, only the pod networks flannel and Weave Net work on multiple architectures.
```

Comment:
Now when we have two options, I'm fine with just saying that "these two providers support ARM, follow their guides for more information"